projects/uitpas/reference/uitpas.json

```
@@ -3079,12 +3082,17 @@
                   "$ref": "#/components/schemas/Pass"
                 },
                 "examples": {
-                  "Example with warning message": {
+                  "Example with expired social tariff and warning message": {
```

```suggestion
                  "Example with active social tariff and warning message": {
```
I think, based on the `"status": "ACTIVE"` below?